static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-support-us-recurring-contributions.js

```diff
@@ -0,0 +1,287 @@
+} from 'common/modules/commercial/contributions-utilities';
+import { setupEpicInLiveblog } from 'common/modules/experiments/tests/acquisitions-epic-liveblog';
+import { viewsInPreviousDays } from 'common/modules/commercial/acquisitions-view-log';
+import {
+    submitInsertEvent,
+    submitViewEvent,
+} from 'common/modules/commercial/acquisitions-ophan';
+import mediator from 'lib/mediator';
+import fastdom from 'lib/fastdom-promise';
+
+import config from 'lib/config';
+
+const baseCampaignCode = 'gdnwb_copts_memco_sandc_us_rec_cont';
+
+const getREFPVID = (): string =>
+    (config.ophan && config.ophan.pageViewId) || 'not_found';
+
+const buildURL = (url: string, campaignCode: string): string =>
+    `${url}${url.indexOf('?') > 0
+        ? '&'
+        : '?'}INTCMP=${campaignCode}&REFPVID=${getREFPVID()}`;
+
+const makeSupportURL = (
+    campaignCode: string,
+    context: boolean = false
+): string =>
+    buildURL(
+        `https://support.theguardian.com/us/contribute${context
+            ? '?context=true'
+            : ''}`,
+        campaignCode
+    );
+
+const makeBecomeSupporterURL = (campaignCode: string): string =>
+    buildURL('https://membership.theguardian.com/us/supporter', campaignCode);
+
+const makeContributionURL = (campaignCode: string): string =>
+    buildURL('https://contribute.theguardian.com/us', campaignCode);
+
+const buildButtonTemplate = ({ supportUrl }) =>
+    `<div class="contributions__amount-field">
+        <div>
+            <a class="contributions__option-button contributions__contribute contributions__contribute--epic contributions__contribute--epic-member contributions__contribute--epic-single-button"
+               href="${supportUrl}"
+               target="_blank">
+                Support the Guardian
+            </a>
+        </div>
+    </div>`;
+
+const liveblogEpicTemplate = (ctaSentence: string) =>
+    `<div class="block block--content is-epic">
+        <p class="block-time published-time">
+            <a href="#" itemprop="url" class="block-time__link">
+                <time data-relativeformat="med" itemprop="datePublished" class="js-timestamp"></time>
+                <span class="block-time__absolute"></span>
+            </a>
+        </p>
+        <div class="block-elements block-elements--no-byline">
+            <p>
+                <em>
+                    Since you’re here &hellip; we have a small favour to ask. More people are reading the Guardian than ever but advertising revenues across the media are falling fast. And <span class="contributions__highlight">unlike many news organisations, we haven’t put up a paywall &ndash; we want to keep our journalism as open as we can</span>. So you can see why we need to ask for your help. The Guardian’s independent, investigative journalism takes a lot of time, money and hard work to produce. But we do it because we believe our perspective matters &ndash; because it might well be your perspective, too.
+                </em>
```

Comment:
Do we need to have the template and copy in the test? We have the Acquisitions copy module for storying copy, and the acquisitions-epic-livegblog.html template, you should be able to implement this using those

Reply:
You can see an example here: 

If you look in acquisitions-epic-liveblog.js, you will see in the control variant the following: 

```template(variant) {
                    return template(liveblogEpicTemplate, {
                        copy: liveblogCopy(
                            variant.options.membershipURL,
                            variant.options.contributeURL
                        ),
                        componentName: variant.options.componentName,
                    });
                },```

Reply:
Changing this doesn't make this test any better / more robust etc. If anything, having it set here protects it from other unintentional changes to the copy made for other reasons.

Reply:
The point of having the templates and copy module is to reduce mistakes, ensure consistency across tests and clean up the test templates. 

While it doesn't make the test "better", it keeps everything tidy and means that test files are less cluttered. By putting the template in here, you are needlessly making the test file bloated and duplicating code. 

Also, above here the button template code is written into the test file, when there is already a template in the project (acquisitions-epic-single-button.html) which would do the job.